Jellyfin.Api/Controllers/TvShowsController.cs

```diff
@@ -86,6 +87,7 @@ public class TvShowsController : BaseJellyfinApiController
         [FromQuery] DateTime? nextUpDateCutoff,
         [FromQuery] bool enableTotalRecordCount = true,
         [FromQuery] bool disableFirstEpisode = false,
+        [FromQuery] bool enableResumable = true,
```

Comment:
Should probably default this to false?

Reply:
I debated that but went with true since it effectively reverts the change that breaks resuming a show when an episode is in progress.

Reply:
I believe this shouldn't break much. We should just keep our eyes out for any bug reports during the 10.9 beta cycle and if this causes issue change the default to `false`.